### PR TITLE
Better error handling for vote constraints

### PR DESCRIPTION
**Pull request by ohenrik**
Added custom error handling for postgresql check constraints on votes >= 0.

### CODE REVIEW

File: app/controllers/topics_controller.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class TopicsController < ApplicationController
+
+  rescue_from PG::CheckViolation, with: :constrian_above_zero
```
Comment (jgorset) [inline app/controllers/topics_controller.rb]: I think you meant to write `constrain` here.

File: app/controllers/topics_controller.rb [hunk 2]
```diff
@@ -95,6 +90,14 @@ def destroy
 
   private
 
+  def constrian_above_zero
```
Comment (jgorset) [inline app/controllers/topics_controller.rb]: … and here, too!

**Comment by jgorset** [comment]
Excellent!!

**Comment by ohenrik** [comment]
I always mess up 'ai' and 'ia' when i type fast :p Corrected it now.

**Comment by jgorset** [comment]
More :cake:!

**Pull request merged**